### ISSUE TITLE
Savvior seem not to like Jquery UI, Chosen dropdown or about any plugin with a similar nurture. 

### ISSUE THREAD

**Issue by LoaiBassam** [attila/savvior#16]
When I use [Jquery UI][2] and/or [Chosen Dropdown][3] inside the `.grid` they just stop working. 

I have put together a test example here: http://loai.directory/test which will demonstrate the bizarre problem live. 

See how nothing like datepickers, dropdowns, accordions and tabs etc work when they are inside the `.grid`? 

I have also set-up datapickers outside the `.grid` so you can compare here: http://loai.directory/test/modules.php

I have been dubbing this for the past few days and I just came one step closer! "I have lost half of my head hair"

Below is the order which my scripts are being called by.

    
    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
    <script type="text/javascript" src="assets/js/jquery-ui.js"></script>
    
    <script type="text/javascript" src="assets/js/main.js"></script>
    <script type="text/javascript" src="assets/js/select-dropdowns.js"></script>
    <script type="text/javascript" src="assets/js/grid-system.js"></script>
    <script type="text/javascript" src="assets/js/form-validation.js"></script>
    <script type="text/javascript" src="assets/js/tooltips.js"></script>
    <script type="text/javascript" src="assets/js/input-masks.js"></script>

I have tried and called grid-system.js just after select-dropdowns.js and the Select Dropdowns suddenly worked! So, I then tried to call grid-system.js at the very end, so after all the scripts are called, but still, only the dropdowns worked, the Jquery UI elements are still broken. 

Even-worse, everything seems to be working perfectly fine on FireFox only...

Please, I have no clue what else I can do, I tried everything I could, your help is much appreciated.

**Comment by LoaiBassam** [comment]
Another test example: http://loai.directory/test - When I place jquery ui elements etc... they do not work inside a .grid - please help

**Comment by attila** [comment]
I believe all your problems could be fixed by using event listeners, see #15
